lib/twinklyMaha_web/live/twinkly_live.ex

```diff
@@ -11,10 +11,15 @@ defmodule TwinklyMahaWeb.TwinklyLive do
   def render(assigns) do
     ~L"""
     <div class="row">
-      <div class="column">
-        <div class="led-box">
-        <div class="led-<%= if @led_on?, do: "green", else: "off" %>"></div>
-        </div>
+      <div class="columns">
+        <%= for n <- 1..8 do %>
+          <%= for m <- 1..8 do %>
+            <div class="led-box">
+            <div class="led led-<%= if @led_on?, do: "on", else: "off" %>" data-ledcolor="purple" phx-hook="LedColor"></div>
```

Comment:
Using the data attributes ([MDN docs](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes)) to send data to our javascript.